docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -271,7 +271,7 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
-## Managing playlists on server side
+## Managing Playlists on Server Side
 
 To manage the media controller playlists in your server application, you must learn to create, save and delete playlists.
```

Comment:
Suggest you to please add a serial comma before and delete:

... create, save, and delete.

Reply:
done